RiotSharp/RiotApi.cs

```diff
@@ -109,255 +110,216 @@ public static RiotApi GetInstance(string apiKey, int rateLimitPer10s, int rateLi
         /// <returns>The instance of RiotApi.</returns>
         public static RiotApi GetInstance(string apiKey, IDictionary<TimeSpan, int> rateLimits)
         {
+            /* todo is this logic required?
             if (instance == null || Requesters.RiotApiRequester == null ||
                 apiKey != Requesters.RiotApiRequester.ApiKey ||
                 !rateLimits.Equals(Requesters.RiotApiRequester.RateLimits))
             {
                 instance = new RiotApi(apiKey, rateLimits);
-            }
-            return instance;
+            } */
```

Comment:
this was meant to get a new API if someone changes api key or rate limits in his code, I don't know if we should keep it